src/test/resources/features/release_version_checksums.feature

```diff
@@ -0,0 +1,130 @@
+Feature: Release Version with checksums
+
+  Background:
+    Given the Consumer has a valid Auth Token
+
+  Scenario: Release binary for Linux 64 bit Platform with checksums
+    Given the existing Default PLATFORM_SPECIFIC java Version is 8u121-zulu
```

Comment:
I would choose a UNIVERSAL candidate, as PLATFORM specific is already covered in the subsequent test.